# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gnupg2/hotfix-ldap-disable-help.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Do not include ldap helper function when ldap support
is disabled at configure time.

  - NoTag <notag@t2sde.org>

--- gnupg-2.4.3/dirmngr/server.c.vanilla	2023-11-20 19:05:10.179805518 +0100
+++ gnupg-2.4.3/dirmngr/server.c	2023-11-20 19:06:07.639802563 +0100
@@ -2776,7 +2776,9 @@
 
   if (opt_help)
     {
+      #if USE_LDAP
       ks_ldap_help_variables (ctrl);
+      #endif
       err = 0;
       goto leave;
     }
